public/data/css.json

```diff
@@ -21,6 +21,19 @@
   {
     "categoryName": "Layouts",
     "snippets": [
+      {
+        "title": "Grid layout",
+        "description": "Equal sized items in a responsive grid",
+        "code": [
+          ".grid-container {",
+          "  display: grid",
+          "  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));",
```

Comment:
Could you add a little comment explaining what values on this line are for ? Like for example what is 250px ? The max or the minimum, what is `auto-fit`, etc